cloud-controller-manager/do/resources.go

```diff
@@ -122,6 +272,7 @@ func (r *ResourcesController) sync() error {
 		return nil
 	}
 
+	tag := buildK8sTag(r.clusterID)
```

Comment:
The `ResourcesController` used to store the cluster ID as tagged version so that we wouldn't need to remember attaching the tag everywhere we'd use it. Was there a specific reason to change that?

Reply:
This change snuck in here when I thought we were returning the droplet ids for nodes in our `GET /v2/kubernetes/cluster/<uuid>` endpoint. I thought we could use that instead of listing all droplets in the account if the cluster id was present. I forgot that I removed droplet id in that api a while back :P. 

One thing that might help is if we built something like a `GET /v2/kubernetes/cluster/<uuid>/resources` endpoint or something that listed all the resources associated with a cluster that we could use here. 🤔 

Should I switch this back for now since it's not used or keep it here?

Reply:
I think it's okay this way too, not a biggie. Was more curious about the reasoning behind it.

Yeah a dedicated API to list resources associated with a cluster would be nice.